.github/workflows/build-test-package-python.yml

```diff
@@ -35,7 +35,7 @@ on:
         description: 'JSON-formatted array of Python 3.x minor version wheel targets'
         required: false
         type: string
-        default: '["8","9","10","11"]'
+        default: '["9","10","11"]'
```

Comment:
Is Python 3.12 supported due to ABI compatibility with 3.11?

Reply:
Yes. And Python 3.13. And Python 3.14 will be, too :-).